src/toxcore/dht/packet/crypto_data.rs

```diff
@@ -163,6 +163,17 @@ impl ToBytes for CryptoDataPayload {
     }
 }
 
+impl CryptoDataPayload {
+    /// Get data skipping zeros at the front
```

Comment:
Why do we need to skip zeros?

Reply:
Because data can be padded with any number of zeros at the beginning. Only after zeros will be packet id and actual payload.

Reply:
Can we do that in packet parsing?

Reply:
Right, just checked - the length of this padding is defined by the length of the data. So we can remove it when parsing and add it back when writing to a byte array.